app/views/shared/_view_my_settings.html.erb

```diff
@@ -37,7 +37,7 @@ See doc/COPYRIGHT.rdoc for more details.
 <h3 class="form--section-title"><%=l(:label_backlogs)%></h3>
 <div class="form--field">
   <%= styled_label_tag :backlogs_task_color, l(:'backlogs.task_color') %>
-  <%= styled_text_field_tag :'backlogs[task_color]', color %>
+  <%= styled_text_field_tag :'backlogs[task_color]', color, container_class: '-middle' %>
```

Comment:
I am not sure if this is the origin of it, but: For inputs from the backlogs plug-in the `container_class: '-middle'` seems to be defined multiple times. Once on the actual container (which is what is supposed to happen) and one more time on the input itself. It doesn't seem problematic to me but maybe this should be addressed for the sake of clarity.